end_to_end/end_to_end_suite_test.go

```diff
@@ -2579,8 +2579,9 @@ LANGUAGE plpgsql NO SQL;`)
 				"--resize-cluster", "--jobs", "3")
 			output, err := gprestoreCmd.CombinedOutput()
 			Expect(err).To(HaveOccurred())
-			Expect(string(output)).To(ContainSubstring(`Error loading data into table public.t1`))
-			Expect(string(output)).To(ContainSubstring(`Error loading data into table public.t2`))
+			Expect(string(output)).To(MatchRegexp(`Error loading data into table public.t1: COPY t1, line \d+, column i: "\d+": ERROR: value "\d+" is out of range for type smallint`))
+			Expect(string(output)).To(ContainSubstring(`Expected to restore 1000000 rows to table public.t2, but restored 0 instead`))
```

Comment:
Isn't it a partial restore?

Reply:
No, partial restore is restore of a part of the table, but here the table was not fully restored.

Reply:
But having the error 'Expected to restore 1000000 rows to table public.t2, but restored 0 instead'  means the COPY itself didn't end with an error, it just got EOF too early. What if we encounter an issue only on one segment, and others are Ok? Willn't we get a situation where we have restored the data on all segments except one?

Reply:
fixed